### ISSUE TITLE
Please release 8.6.1 to pypi

### ISSUE THREAD

**Issue by blackside** [domainaware/parsedmarc#421]
Dont know how releases to pypi are handled but thought I would make a mention that Im hoping for it to show up soon.

**Comment by jippi** [comment]
I wasted a good couple of hours trying to debug the ES bug until I found this issue :) 

Please release it to pypi :)

**Comment by wmax641** [comment]
For anyone else banging their head on the keyboard with ElasticSearch issues,

This will fix issue #410 

For when `parsedmarc` tries to run `save_aggregate_report_to_elasticsearch()`, but the `elasticsearch` python module fails with this exception: 

```
RequestError(400, 'x_content_parse_exception', '[1:59] [bool] failed to parse field [must]')
```

**Comment by seanthegeek** [comment]
Done. I included PR #417 too 😊
